package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "cockpit",
+  "productName": "Cockpit",
```

Comment:
Shouldn't this be in `build` ?

Reply:
[Both work](https://www.electron.build/configuration/configuration#configuration).

Reply:
I would just recommend moving it to build to make it more clear that's for electron.
Over in the global scope.

Reply:
Done.